src/licenses/config.rs

```diff
@@ -97,8 +97,19 @@ pub struct KrateConfig {
     pub ignore: Vec<Ignore>,
 }
 
+#[serde(rename_all = "kebab-case")]
 #[derive(Deserialize, Debug, Default)]
 pub struct Config {
+    /// Only includes dependencies that match at least
+    /// one of the specified targets
+    #[serde(default)]
+    pub targets: Vec<String>,
+    /// Ignores any build dependencies in the graph
+    #[serde(default)]
+    pub ignore_build_dependencies: bool,
+    /// Ignores any dev dependencies in the graph
+    #[serde(default)]
+    pub ignore_dev_dependencies: bool,
```

Comment:
Can you add those docs also to the readme?

Reply:
Yup!